site/docs/components/data.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Data - Typography
+navTitle: Data - Typography
```

Comment:
This is a stop-gap name for now. If we went with `<Typography.Data>` down the track it still doesn't solve the name, so we'll need to think about that.